Makefile

```diff
@@ -73,11 +73,7 @@ gosec-scan:
 
 .PHONY: build
 build:
-	@build/common/scripts/gobuild.sh build/_output/bin/$(IMG) ./
-
-.PHONY: local
-local:
-	@GOOS=darwin build/common/scripts/gobuild.sh build/_output/bin/$(IMG) ./cmd/manager
+	@go build -o build/_output/bin/$(IMG) ./
```

Comment:
Why hide the command with `@`?

Reply:
I was just following what was done before.